src/abstractdataframe/reshape.jl

```diff
@@ -582,12 +582,16 @@ with name specified by `dest_namescol`.
 # Arguments
 - `df` : the `AbstractDataFrame`
 - `src_namescol` : the column that will become the new header.
-  This column's element type must be `AbstractString` or `Symbol`.
 - `dest_namescol` : the name of the first column in the returned `DataFrame`.
   Defaults to the same name as `src_namescol`.
 - `makeunique` : if `false` (the default), an error will be raised
   if duplicate names are found; if `true`, duplicate names will be suffixed
   with `_i` (`i` starting at 1 for the first duplicate).
+- `strict` : if `true` (the default), an error will be raised if the values
+  contained in the `src_namescol` are not all `Symbol` or all `AbstractString`,
+  or can all be converted to `AbstractString` using `convert`. If `false`
+  then any values are accepted and the will be changed to strings using
+  the `string` function.
```

Comment:
```suggestion
  the [`string`](@ref) function.
```

Reply:
@nalimilan would such reference to Base Julia work?

CC @mortenpi

Reply:
(I have accidentally committed this suggestion, but the question remains)

Thank you!

Reply:
OK - no need to answer - CI gave me the answer. I revert the change.

Reply:
Ah yes sorry that doesn't work indeed. So my net contribution to this PR is clearly negative...